ide-support/revliburl.livecodescript

```diff
@@ -73,6 +80,49 @@ on libraryStack
    initialiseCustomProps
 end libraryStack
 ------------------------------------------------
+-- CW-2016-06-11: [[ External driver support ]] Add function to select external driver for use with libUrl.
+## Set driver to use for libUrl
+on libUrlSetDriver pDriver
```

Comment:
@montegoulding: This is what we want to solve with the 'extension' format, there's no reason an extension shouldn't include multiple 'things' which get slotted into the right place both in the IDE and in a Standalone... However, right now we don't have that infrastructure, so we'll probably have to 'special-case' this in the S/B... @livecodeali might have some better ideas!